sql/updates/world/3.3.5/2016_05_15_15_world.sql

```diff
@@ -0,0 +1,52 @@
+-- Bunthen Plainswind (NPC 11798) and Silva Fil'naveth (NPC 11800), Moonglade Druid Flight Masters
+UPDATE `creature_template` SET `AIName`= 'SmartAI', `ScriptName`= '' WHERE `entry` IN (11798,11800);
+
+DELETE FROM `gossip_menu_option` WHERE `menu_id` IN (4041,4042);
+INSERT INTO `gossip_menu_option` (`menu_id`,`id`,`option_icon`,`option_text`,`OptionBroadcastTextID`,`option_id`,`npc_option_npcflag`,`action_menu_id`,`action_poi_id`,`box_coded`,`box_money`,`box_text`,`BoxBroadcastTextID`) VALUES
+(4041,0,0,"I'd like to fly to Rut'theran Village.",                             7573,1,1,   0,0,0,0,'',0),
+(4041,1,0,"Do you know where I may find the Half Pendant of Aquatic Agility?",  8035,1,1,4225,0,0,0,'',0),
+(4041,2,0,"Do you know where I may find the Half Pendant of Aquatic Agility?",  8035,1,1,4226,0,0,0,'',0),
+(4042,0,0,"I'd like to fly to Thunder Bluff.",                                 12804,1,1,   0,0,0,0,'',0),
+(4042,1,0,"Do you know where I may find the Half Pendant of Aquatic Endurance?",8036,1,1,4223,0,0,0,'',0),
+(4042,2,0,"Do you know where I may find the Half Pendant of Aquatic Endurance?",8036,1,1,4224,0,0,0,'',0);
+
+-- Link gossip_menu_option.menu_id (gossip_menu.entry) to npc_text.id (gossip_menu.text_id)
+DELETE FROM `gossip_menu` WHERE `text_id` IN (4917,4918,5373,5374,5375,5376);
+INSERT INTO `gossip_menu` (`entry`,`text_id`) VALUES
+(4042,4917),
+(4042,4918),
+(4223,5373),
+(4224,5374),
+(4225,5375),
+(4226,5376);
```

Comment:
Here is how I looked up the most likely place to pick those `gossip_menu`.`entry` values:
```sql
SELECT * FROM `gossip_menu` WHERE `text_id` BETWEEN 5300 AND 5400;
```
`gossip_menu` content before my changes:
```
 entry  text_id
------  -------
  4203     5320
  4204     5325
  4205     5330
  4207     5340
  4208     5345
  4209     5350
  4210     5355
  4211     5360
  4241     5397
  4242     5398
  4842     5335
```
`gossip_menu` content after my changes:
```
 entry  text_id
------  -------
  4203     5320
  4204     5325
  4205     5330
  4207     5340
  4208     5345
  4209     5350
  4210     5355
  4211     5360
  4223     5373
  4224     5374
  4225     5375
  4226     5376
  4241     5397
  4242     5398
  4842     5335
```